### PR TITLE
feat(block): adding raw block volume support for LVM

### PR DESCRIPTION
**Pull request by pawanpraka1**
Signed-off-by: Pawan <pawan@mayadata.io>

**Why is this PR required? What issue does it fix?**:

adding raw block volume support for LVM

**Does this PR require any upgrade changes?**:

No

**If the changes in this PR are manually verified, list down the scenarios covered:**:

tested with the sample fio yaml added in this PR

### CODE REVIEW

File: pkg/driver/agent.go [hunk 1]
```diff
@@ -115,7 +115,8 @@ func (ns *node) NodePublishVolume(
 	// If the access type is block, do nothing for stage
```
Comment (akhilerm) [inline pkg/driver/agent.go:115]: Didnt get this comment. This is the publish request.

Comment (pawanpraka1) [reply pkg/driver/agent.go:115]: this is not a vaild comment here, this is for nodestagevolume grpc call.

File: pkg/lvm/mount.go [hunk 2]
```diff
@@ -199,3 +199,32 @@ func MountFilesystem(vol *apis.LVMVolume, mount *MountInfo) error {
...
 	return MountVolume(vol, mount)
 }
+
+// MountBlock mounts the block disk to the specified path
+func MountBlock(vol *apis.LVMVolume, mountinfo *MountInfo) error {
+	target := mountinfo.MountPath
+	volume := vol.Spec.VolGroup + "/" + vol.Name
+	devicePath := DevPath + volume
+
+	mountopt := []string{"bind"}
```
Comment (akhilerm) [inline pkg/lvm/mount.go:209]: Why do we need to bind mount the device at a path during nodepublish. Is it a requirement from CSI?

**Review by akhilerm — COMMENTED** [review]
given a comment regarding workflow.

File: pkg/lvm/mount.go (hunk 2 above)
Comment (pawanpraka1) [reply pkg/lvm/mount.go:209]: yes, we need to moount the device to the path where kubelet wants us to.

File: pkg/driver/agent.go (hunk 1 above)
Comment (akhilerm) [reply pkg/driver/agent.go:116]: @pawanpraka1  Can you remove this comment,

**Review by akhilerm — APPROVED** [review]
LGTM. Gave a suggestion on removing the comment.

**Review by kmova — APPROVED** [review]

**Pull request merged**